tools.py

```diff
@@ -105,6 +105,7 @@ def onPick(self, event):
         '''Get value at the point selected by mouse click.'''
         xdata = event.xdata # get event x location
         ydata = event.ydata # get event y location
+        #XXX this fails if not inside plot region
```

Comment:
picking outside the plot region generate an error in the terminal, how to avoid this error?